howto/howto-insert-contacts.md

```diff
@@ -151,10 +151,10 @@ To include only the given set of fields (data) in each of the insert operation,
 .include(fields)
 ```
 
-For example, to only include email fields,
+For example, to only include email and name fields,
 
 ```kotlin
-.include(Fields.Email.all)
+.include { Email.all + Name.all }
```

Comment:
Updated all docs to include this to make sure folks understand the syntactic beauty of Kotlin 😄